internal/transport/controlbuf.go

```diff
@@ -773,7 +774,7 @@ func (l *loopyWriter) earlyAbortStreamHandler(eas *earlyAbortStream) error {
 	}
 
 	headerFields := []hpack.HeaderField{
-		{Name: ":status", Value: "200"},
+		{Name: ":status", Value: strconv.Itoa(int(eas.httpStatus))},
```

Comment:
Should we now validate `eas.httpStatus != 0` in case a caller forgets to set it?  Put an error in the log in that case and tests will fail if it ever happens.  Or should we say `if httpStatus == 0 { httpStatus = 200 }`?  I'm OK with either, but I think we should do something as defensive programming.

Reply:
Eh, I don't really think we need to be defensive against something that is called only at a few places and entirely internally. I added the 0 -> 200 switch though.